src/domain/synthetics/trade/utils/swapStats.ts

```diff
@@ -19,16 +19,41 @@ export function getSwapPathOutputAddresses(p: {
   swapPath: string[];
   wrappedNativeTokenAddress: string;
   shouldUnwrapNativeToken: boolean;
+  isIncrease: boolean;
 }) {
-  const { marketsInfoData, initialCollateralAddress, swapPath, wrappedNativeTokenAddress, shouldUnwrapNativeToken } = p;
+  const {
+    marketsInfoData,
+    initialCollateralAddress,
+    swapPath,
+    wrappedNativeTokenAddress,
+    shouldUnwrapNativeToken,
+    isIncrease,
+  } = p;
 
   if (swapPath.length === 0) {
-    return {
-      outTokenAddress:
-        shouldUnwrapNativeToken && initialCollateralAddress === wrappedNativeTokenAddress
-          ? NATIVE_TOKEN_ADDRESS
-          : initialCollateralAddress,
+    // If the swap path is empty and is increase order,
+    // we do not care about the output token address during happy path,
```

Comment:
this line doesnt make sense to me, could you pls rephrase it?

Reply:
rephrased to 

```
      // During increase target collateral token is always ERC20 token, it can not be native token.
      // Thus we do not need to check if initial collateral token is wrapped token to unwrap it.
      // So we can safely return initial collateral token address as out token address, when there is no swap path.
```